devicetypes/smartthings/nyce-motion-sensor.src/nyce-motion-sensor.groovy

```diff
@@ -13,6 +13,7 @@
  *  for the specific language governing permissions and limitations under the License.
  *
  */
+import physicalgraph.zigbee.clusters.iaszone.ZoneStatus
```

Comment:
i meant when you do a zigbee.parse, it creates an object of the zigbee class. if the zigbee class imports the zone status, then the DTH wouldn't need to do it. you might have to define the object as 'def' to avoid the error.